Automattic/newspack-theme
# fix: update prettier settings to remove deprecated check

### All Submissions:




### Changes proposed in this Pull Request:

This PR updates a deprecated linting function, replacing `jsxBracketSameLine` with `bracketSameLine`. This is to resolve some linting errors thrown by CircleCI being thrown for PRs as of https://github.com/Automattic/newspack-theme/pull/1836.

### How to test the changes in this Pull Request:

1. Run `npm run lint` in the theme directory. 
2. Note the following errors:

```
jsxBracketSameLine is deprecated.
```

and

```
...wp-content/themes/newspack-theme/js/src/post-meta-toggles.js
  18:9  error  Replace `⏎↹↹newspack_hide_page_title,⏎↹↹newspack_hide_updated_date,⏎↹↹newspack_show_share_buttons,⏎↹}·=·` with `·newspack_hide_page_title,·newspack_hide_updated_date,·newspack_show_share_buttons·}·=⏎↹↹`  prettier/prettier`.
```

3. Apply the PR. 
4. Confirm that `npm run lint` no longer returns errors. 

### Other information:
